### ISSUE TITLE
Plugin release tool: did not open the editor and canceled the process

### ISSUE THREAD

**Issue by oandregal** [WordPress/gutenberg#27872]
I've just released Gutenberg 9.6 using the release tool `./bin/cli.js release`. The only issue I had was in this step:

```md
? Please provide the CHANGELOG of the release (markdown) Press <enter> to launch your preferred editor.
```

The editor wouldn't open and the process was canceled. To circumvent this, I've inlined the changelog by going [here](https://github.com/WordPress/gutenberg/blob/master/bin/plugin/commands/release.js#L669) and substituting:

```js
	const { changelog } = await inquirer.prompt( [
		{
			type: 'editor',
			name: 'changelog',
			message: 'Please provide the CHANGELOG of the release (markdown)',
		},
	] );
```

by 

```js
const changelog = `
### changelog section

- items 
- ...

`;
```

## Info about my system

I'm listing here some potential info for debugging:

- OS: I'm on a Linux system, Ubuntu 20.10.
- Terminal: gnome-terminal 3.38.0
- The `$EDITOR` env variable is set to `/usr/bin/nano` and I've tested that the editor works fine.

## How to repro the issue

Do not worry, you can stop the process before anything is published.

1. Run the `./bin/plugin/cli.js release` command.
2. Say yes to the first message.
3. When you're shown the `? Please provide the CHANGELOG of the release (markdown) Press <enter> to launch your preferred editor.` message, press ENTER.
4. The expected result is that it opens an editor and when you save&close it goes back to the process.
5. Cancel after this step.

Whether you can or can't repro the issue it'd be valuable if you can share your setup anyway.

**Comment by nosolosw** [comment]
The [docs](https://www.npmjs.com/package/inquirer#editor---type-editor) mention that [inquirer](https://www.npmjs.com/package/inquirer#editor---type-editor), the library we use for command-line interaction, uses `$ENVIRONMENT` or `$VISUAL` instead of `$EDITOR`. I've tried to set up all of them but still didn't work.

The docs also say that it uses `vim` as a fallback. I tried installing vim and I can confirm the fallback works.
